addons/mail/static/src/js/views/activity/activity_model.js

```diff
@@ -2,12 +2,20 @@ odoo.define('mail.ActivityModel', function (require) {
 'use strict';
 
 var AbstractModel = require('web.AbstractModel');
+var session = require('web.session');
 
 var ActivityModel = AbstractModel.extend({
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+    * @override
+    */
+    init: function (parent) {
+        this._super.apply(this, arguments);
+        this.context = parent.userContext.length ? parent.userContext : session.user_context || {};
```

Comment:
I think you just need `session.user_context`. It looks like `userContext` is always computed from `session.user_context`.